libs/langchain/langchain/vectorstores/neo4j_vector.py

```diff
@@ -263,6 +300,39 @@ def retrieve_existing_index(self) -> Optional[int]:
         except IndexError:
             return None
 
+    def retrieve_existing_fts_index(self) -> Optional[str]:
+        """
+        Check if the fulltext index exists in the Neo4j database
+
+        This method queries the Neo4j database for existing fts indexes
+        with the specified name.
+
+        Returns:
+            (Tuple): keyword index information
+        """
+
+        index_information = self.query(
+            "SHOW INDEXES YIELD name, type, labelsOrTypes, properties, options "
+            "WHERE type = 'FULLTEXT' AND (name = $keyword_index_name "
+            "OR (labelsOrTypes = [$node_label] AND "
```

Comment:
that might not be true, better use $node_label IN labelsOrTypes (because FTS can be multi-label)

Reply:
Yes, but then I don't want to use that one if it is multi-labeled or has multiple-properties?